### DISCUSSION TITLE
求助：通过官网的使用方式，在我Demo里无法展示

### DISCUSSION THREAD

**Discussion by Wingle** [Tencent/libpag#1671] [Q&A]
通过官网的使用方式，在我Demo里无法展示。
代码如下：
`- (void)viewDidLoad
{
    [super viewDidLoad];
    
    NSString *bundlePath = [[NSBundle mainBundle] pathForResource:@"replacement" ofType:@"pag"];
    PAGFile *pagFile = [PAGFile Load:bundlePath];

    // 此时pagFile 非nil
    
    self.pagView = [[PAGView alloc] initWithFrame:self.view.bounds];
    [self.view addSubview:self.pagView];
    [self.pagView setComposition:pagFile];
    [self.pagView setRepeatCount:0];
    [self.pagView play];
}`

没看到有什么log 输出，无法定位，求帮忙解答下。

**Comment by Wingle** [comment]
版本问题

**Reply by kevingpqi123** [reply]
此类问题和 pag 的 demo 工程对比就可以了
